src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -98,6 +109,33 @@ internal static class ParameterHelpers
             return parameters;
         }
 
+        private static bool IsPropertyWithoutBody(Symbol owner)
+        {
+            if (owner is SourcePropertySymbol)
```

Comment:
> Should the method return true if the symbol is property, but not SourcePropertySymbol?

What is the case where this can happen?  When can you have parameters on a property but it isn't a SourcePropertySymbol?